server/routes/index.ts

```diff
@@ -298,6 +298,34 @@ export function defineRoutes(router: IRouter, esClient: IClusterClient) {
       }
     }
   );
+
+  router.put(
```

Comment:
Should be unified as `POST`.

Reply:
Sure. Updated to POST to be consistent with other APIs. I only keep the place where it interacts with ES to be PUT as this is an ES side requirement.